spec/classes/lightblue/application/datamgmt_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+describe 'lightblue::application::datamgmt' do
+  let(:hiera_config){ 'spec/fixtures/hiera/hiera.yaml' }
+
+  let :facts do
+    {
+      :architecture => 'x86_64',
+      :osfamily => 'RedHat',
+      :operatingsystemrelease => '7.0'
+    }
+  end
+
+  context 'with default params' do
+    it do
+      should contain_package('lightblue-data-mgmt') \
+        .with_ensure('latest') \
+        .with_require(['Class[Lightblue::Yumrepo::Lightblue]', 'Class[Lightblue::Eap]'])
+    end
+
+    it do
```

Comment:
IMHO splitting the asserts across multiple test cases is a good thing. For instance, there are 3 asserts. Say two of them will fail. If I combine all 3 to one `it` block, then I will get one test failure, with one failure message -- the other failing assert will be hidden until I fix the first one and rerun the tests.

If I separate them into 3 separate `it` blocks, I see two tests fail and two failure messages, and can fix them all before running the tests again.

I googled for a few best practices guides with RSpec, and it looks like they agree for the most part:
- https://github.com/reachlocal/rspec-style-guide#good-example-6
- http://betterspecs.org/#single

Personally, I have no problems deferring to your preference if you would like, I just wanted to explain what I thought were the advantages of isolated the assertions.